README.md

```diff
@@ -7,11 +7,11 @@
 [![latest release](https://img.shields.io/github/v/release/terraform-ibm-modules/terraform-ibm-kms-all-inclusive?logo=GitHub&sort=semver)](https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/releases/latest)
 [![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
 
-This module combines the following Key Protect modules to create a full end-to-end key infrastructure:
+This module combines the following key management service (KMS) modules to create a full end-to-end key infrastructure:
 
 - [Key Protect module](https://github.com/terraform-ibm-modules/terraform-ibm-key-protect)
-- [Key Protect key module](https://github.com/terraform-ibm-modules/terraform-ibm-key-protect-key)
-- [Key Protect key ring module](https://github.com/terraform-ibm-modules/terraform-ibm-key-protect-key-ring)
+- [KMS Key module](https://github.com/terraform-ibm-modules/terraform-ibm-kms-key)
+- [KMS Key Ring module](https://github.com/terraform-ibm-modules/terraform-ibm-kms-key-ring)
 
 The module takes a map, called `key_map`, that supports hierarchical "key rings" for a single key management service (KMS) instance. Because access to key rings is managed in the KMS, you can comply with controls around least privilege (for example, [NIST AC-6](https://csrc.nist.gov/Projects/risk-management/sp800-53-controls/release-search#/control?version=4.0&number=AC-6)) and can reduce the number of access groups you need to assign. For more information about key rings, see [Grouping keys together using key rings](https://cloud.ibm.com/docs/key-protect?topic=key-protect-grouping-keys).
```

Comment:
`key_map` reference needs to be updated here. We no longer use a map, we use a `keys` list object right?